actionview/lib/action_view/template.rb

```diff
@@ -202,8 +202,12 @@ def refresh(view)
       end
     end
 
+    def short_identifier
+      @short_identifier ||= defined?(Rails.root) ? identifier.sub("#{Rails.root}/", "") : identifier
```

Comment:
I’m guessing what’s identifier and short_identifier here is what’s path(?) and virtual_path elsewhere in Action View. Might be nice if the naming was unified for our internals. Though that’s another PR.

Reply:
I think they are a little different. `virtual_path` is relative to the view directory (ex. `app/views`), where `short_identifier` is relative to the app root (or from `/` if from a gem outside). I think it's also possible for identifier not to be from the FS if the template comes from some other source.

I do agree that it does feel like something here could be unified though!